websites/S/SoundCloud/presence.ts

```diff
@@ -156,7 +156,7 @@ presence.on("UpdateData", async () => {
   const playButton = document.querySelector(".playControls__play.playing");
   const playing = playButton ? true : false;
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Combine this with the previous 'const' statement. [one-var](https://eslint.org/docs/rules/one-var)